content/docs/communities/moderation/_index.md

```diff
@@ -95,21 +93,280 @@ in the message bar at the bottom:
 /op @yourMjolnirBot:example.com 100
 ```
 
-TODO screenshot of oping mjolnir
+The bot really does need those privileges to be able to actually kick and ban
+people from the rooms it protects, or change other people's power level. Keep in
+mind that whoever has access to Mjolnir credentials effectively has powerlevel
+100 in those rooms. In this configuration Mjolnir is de-facto the owner of your
+community rooms. Some more advanced configurations allow you to keep more
+privileges than mjolnir, but they are quite tedious to set-up. We will not cover
+in this documentation.
 
 ## Getting moderation reports
 
-- You need your own instance while Aristotle isn't merged in the spec and
-  deployed in the wild?
+By default, the moderation reports go to the reporter's homeserver
```

Comment:
I fear it won't get updated again but maybe add a link to the msc element Web implements that sends them to mods instead?